test/modules/core/lib/pick-layers.spec.js

```diff
@@ -54,12 +54,12 @@ const NEW_GRID_LAYER_PICK_METHODS = {
     {
       parameters: {
         x: 300,
-        y: 300
+        y: 209
       },
       results: {
         count: 1,
         // point count in the aggregated cell for each pickInfo object
-        cellCounts: [1]
+        cellCounts: [8]
```

Comment:
@1chandu Not sure why `cellCounts` changed...

Reply:
`y` is changed from `300` to `209`, so must be picking a different cell?

Reply:
See PR description - we were mistakenly picking the pixel above the specified position. After the fix, I shifted the test case params one pixel up so they are picking the same pixel as before.
Technically speaking this test case is picking a different subpixel (device pixel ratio is 2). From visual inspection I believe they belong to the same cell. Unless picking results differ by pixel there is something else wrong.